src/model.rs

```diff
@@ -297,17 +257,20 @@ impl Hashable<BlockHash> for Block {
         _ = write!(hasher, "{}", self.block_parent_hash);
         _ = write!(hasher, "{}", self.block_height);
         _ = write!(hasher, "{}", self.block_timestamp);
-        for tx in self
-            .new_contract_txs
-            .iter()
-            .chain(self.new_blob_txs.iter())
-            .chain(self.new_verified_proof_txs.iter())
-        {
+        for tx in self.new_contract_txs.iter().chain(self.new_blob_txs.iter()) {
             hasher.update(tx.hash().0);
         }
-        for (tx_hash, blob_v) in self.verified_blobs.iter() {
+        /*
+        TODO: is this redundant?
+        for tx in self.new_verified_proof_txs.iter() {
+            _ = write!(hasher, "{}", tx.);
+            _ = write!(hasher, "{}", blob_v);
+        } */
```

Comment:
Can we have new verified proof that verify no blobs ? I don't think so, those will be rejected I guess ? So I think we can avoid having them in hash

Reply:
I think we should actually just remove the hashing of Block entirely and #481 does it, so the question is a bit moot.
Need to look at it in that context